### PR TITLE
Bump the version to 0.1.4.

### PR DESCRIPTION
**Pull request by SamirTalwar**

### CODE REVIEW

**Review by soupi — APPROVED** [review]

File: ndc-reference/bin/reference/main.rs [hunk 1]
```diff
@@ -2353,7 +2353,7 @@ mod tests {
 
             let mut expected = mint.new_goldenfile(expected_path).unwrap();
 
-            let response_json = serde_json::to_string_pretty(&response.0).unwrap();
+            let response_json = serde_json::to_string_pretty(&response.0).unwrap() + "\n";
```
Comment (paf31) [inline ndc-reference/bin/reference/main.rs:2356]: Is this deliberate?

**Comment by SamirTalwar** [comment]
I'm putting this into draft until we decide it's a good idea to release a new version.

File: ndc-reference/bin/reference/main.rs (hunk 1 above)
Comment (paf31) [reply ndc-reference/bin/reference/main.rs:2356]: We shouldn't be editing these by hand really, though. You can run `UPDATE_GOLDENFILES=1 cargo test` to update golden files when there is a breaking change.

Comment (SamirTalwar) [reply ndc-reference/bin/reference/main.rs:2356]: I have had trouble getting that command to work. (I highly recommend we switch to [Insta](https://insta.rs/).)

**Pull request closed without merging**